README.md

```diff
@@ -65,3 +67,22 @@ BB_SERVER_LOCATION <url>
 FHIR_PAYLOAD_DIR <directory_path>
 BB_TIMEOUT_MS <integer>
 ```
+
+### Other things you can do
+
+Use docker to look at the api database with psql:
+```sh
+docker run --rm --network bcda-app_default -it postgres psql -h bcda-app_db_1 -U postgres bcda
```

Comment:
Thank you for adding this database info! I've had to do it a hundred times and never thought to document it.